android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -141,6 +141,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 }
                 sendBroadcast(new Intent(BookmarkWidgetProvider.ACTION_UPDATE));
                 return true;
+
+            case R.id.action_share:
+                Intent sendIntent = new Intent();
+                sendIntent.setAction(Intent.ACTION_SEND);
+                sendIntent.putExtra(Intent.EXTRA_TEXT, "essetial info to share");
```

Comment:
Please move this to `strings.xml` instead of hard-coding the values.
It'll help us later on while translating the app.